app/views/catalog/_show_default.html.erb

```diff
@@ -0,0 +1,8 @@
+<dl class="dl-horizontal  dl-invert">
+  <% document_show_fields(document).each do |field_name, field| -%>
+    <% unless document.field_value_is_uri(field_name).empty? %>
```

Comment:
Im thinking about adding a new method to document to do this instead of checking to see if what is returned by #field_value_is_uri is an empty array.